third_party/bazel/src/java_tools/buildjar/java/com/google/devtools/build/buildjar/jarhelper/BUILD.bazel

```diff
@@ -0,0 +1,13 @@
+# Copyright 2014 The Bazel Authors. Licensed under Apache License, Version 2.0
+load("@rules_java//java:defs.bzl", "java_library")
+
+licenses(["notice"])
+
+java_library(
+    name = "jarhelper",
+    srcs = [
```

Comment:
I just copied this out of the upstream BUILD.bazel file, so I'll elave it as it to avoid causing someone to think we've modified the source. https://github.com/bazelbuild/bazel/blob/master/src/java_tools/buildjar/java/com/google/devtools/build/buildjar/jarhelper/BUILD

Reply:
Ah, I forgot about that. That makes sense.